DonjinKrawler-Server/src/main/java/donjinkrawler/Server.java

```diff
@@ -1,5 +1,7 @@
 package donjinkrawler;
 
+import donjinkrawler.enemy_generation.Enemy;
```

Comment:
[nitpick]
Maybe change the package name to something less verbose? enemies? mobs? entities?